internals/src/macros.rs

```diff
@@ -37,3 +37,25 @@ macro_rules! impl_to_hex_from_lower_hex {
         }
     };
 }
+
+/// Quick and dirty macro for parsing hex in tests.
+///
+/// For improved ergonomics import with: `use hex_lit::hex;`
+#[macro_export]
+#[cfg(feature = "alloc")]
+#[cfg(feature = "hex")]
+macro_rules! test_hex_unwrap (($hex:expr) => (<alloc::vec::Vec<u8> as hex::FromHex>::from_hex($hex).unwrap()));
+
+#[cfg(test)]
+#[cfg(feature = "alloc")]
+#[cfg(feature = "hex")]
+mod tests {
+    #[test]
+    fn parse_hex_into_vector() {
```

Comment:
Is this still needed?

Reply:
The test you mean or the macro itself? The macro is used and I'd say if we use the macro then we keep the test as well.

Reply:
I meant the macro, yes. Interesting that it's used - we have dynamic test data somewhere.